data-access-analysis-tools/portal-intro.rst

```diff
@@ -148,75 +179,79 @@ The language is used by the `IVOA <https://ivoa.net>`_ to represent astronomy qu
-In the `truth_match` table, ``is_good_match`` is ``true`` (or ``1``) if an object-truth matching pair satisfies all matching criteria, or it is ``false``(or ``0``) otherwise.
-``is_good_match`` for an object is defined as, separations <1 arcsec and magnitude differences <1 mag.
-This reduces the number of results returned from 205 to 191.
-
-.. figure:: /_static/portal_results_join.png
-    :name: portal_results_join
-    :width: 600
-
-    The results of a JOIN.
-
-Note that ``is_good_match`` is of type boolean whereas in the ADQL query above we selected good matches by filtering on ``truth.is_good_match`` = ``1`` . With ADQL, the "=0" (false) or "=1" (true) syntax for booleans should be used.
-
-**Query the TAP service schema**
+   SELECT src.ccdVisitId, src.extendedness, src.band,
+   scisql_nanojanskyToAbMag(src.psfFlux) AS psfAbMag,
+   cv.obsStartMJD, cv.seeing 
+   FROM dp02_dc2_catalogs.Source AS src
+   JOIN dp02_dc2_catalogs.CcdVisit AS cv
+   ON src.ccdVisitId = cv.ccdVisitId
+   WHERE CONTAINS(POINT('ICRS', coord_ra, coord_dec), CIRCLE('ICRS', 62.0, -37, 1)) = 1
+   AND src.band = 'i' AND src.extendedness = 0 AND src.psfFlux > 10000
+   AND cv.obsStartMJD > 60925 AND cv.obsStartMJD < 60955
+    
+**Query the TAP service schema**:
 Information about the LSST TAP schema can also be obtained via ADQL queries.
 The following query gets the names of all the available DP0.2 tables.
 
 .. code-block:: SQL
 
    SELECT *
    FROM tap_schema.tables
-   WHERE tap_schema.tables.table_name like 'dp01%'
+   WHERE tap_schema.tables.table_name like 'dp02%'
 
-To get the detailed list of columns available in the `object` table, their associated units and descriptions:
+To get the detailed list of columns available in the "Object" table, their associated units and descriptions:
 
 .. code-block:: SQL
 
    SELECT tap_schema.columns.column_name, tap_schema.columns.unit,
    tap_schema.columns.description
    FROM tap_schema.columns
-   WHERE tap_schema.columns.table_name = 'dp01_dc2_catalogs.object'
-
-This produces a subset of the data shown in the lower-right pane of the Portal's **Single Table** query screen, described above.
+   WHERE tap_schema.columns.table_name = 'dp02_dc2_catalogs.Object'
 
 See also :ref:`DP0-2-Tutorials-Portal` for additional demonstrations of how to use the Portal's ADQL functionality.
+
+
+.. _Portal-Intro-Image-Queries:
+
+Image Queries
+=============
+
+This is a placeholder for further instructions about how to use the image query interface and how to interact with the search results in the Portal.
+
+These two figures are representative of what this functionality will include.
+
+.. figure:: /_static/portal_ImageQueryDP02.png
+    :name: portal_ImageQueryDP02
+
+    An early version of the Portal's image query interface.  
+
+
+.. figure:: /_static/portal_ImageQueryCoverageDP02.png
+    :name: portal_ImageQueryCoverageDP02
+
+    An early version of the Portal's image query search results coverage map.  
```

Comment:
Not sure you want to add any more, but I think that this image is showing the outlines of a coadd patch (orange) and calexps (pink) that cover a queried point.

Reply:
Oh that was just a really early placeholder, I added much much more.